internal/signerverifier/ssh/ssh.go

```diff
@@ -56,10 +60,14 @@ func (v *Verifier) Public() crypto.PublicKey {
 	return v.sshKey.(ssh.CryptoPublicKey).CryptoPublicKey()
 }
 
+func (v *Verifier) MetadataKey() *sv.SSLibKey {
+	return v.sslibKey
+}
+
 // Signer is a dsse.Signer implementation for SSH keys.
 type Signer struct {
-	Verifier *Verifier
-	Path     string
+	Path string
+	*Verifier
```

Comment:
This way, Signer inherits Verifier's methods.

Reply:
Ah, right, it's the embedding thing. Had forgotten about that.